tests/test_sim.py

```diff
@@ -516,6 +516,45 @@ def process():
                 self.assertEqual((yield self.o), 0)
             sim.add_sync_process(process)
 
+    def setUp_clock_phase(self):
+        self.m = Module()
+        self.m.domains.phase0 = ClockDomain()
```

Comment:
Style I was intending is: `self.phase0 = m.domains.phase0 = ClockDomain()`

Reply:
Ah, I should have asked, I guess. So, this is the way =) Will fix.